Tests/mock_server.py

```diff
@@ -223,6 +223,7 @@ def configure_proxy_in_demisto(username, password, server, proxy=''):
         }
         response = demisto_client.generic_request_func(self=client, path='/system/config',
```

Comment:
note `response` contains more than just the server conf so the log below might be a bit misleading